Update install_openjdk6.bat due to deprecated url & jdk

parzival0037 opened the pull request:
Update install_openjdk6.bat due to deprecated url & jdk6 to jdk7. The vagrant .\build1.ps1 script fails due to this install_openjdk6.bat script failing. The file was not updated in the past 5 years and the jdk repo URL has changed by alexkasko Link to repo: https://github.com/alexkasko/openjdk-unofficial-builds/.

The file itself is present on bitbucket itself now. Link to the latest repo: https://bitbucket.org/alexkasko/openjdk-unofficial-builds/downloads/openjdk-1.7.0-u80-unofficial-windows-amd64-installer.zip

Fixed the batch script to latest url. The script dosen't fail now.

parzival0037 committed:
Update install_openjdk6.bat due to deprecated url & jdk

Update install_openjdk6.bat due to deprecated url & jdk6 to jdk7.
The vagrant .\build1.ps1 script fails due to this install_openjdk6.bat script failing. The file was not updated in the past 5 years and the jdk repo URL has changed by alexkasko Link to repo: https://github.com/alexkasko/openjdk-unofficial-builds/.

The file itself is present on bitbucket itself now. Link to the latest repo: https://bitbucket.org/alexkasko/openjdk-unofficial-builds/downloads/openjdk-1.7.0-u80-unofficial-windows-amd64-installer.zip

Fixed the batch script to latest url. The script dosen't fail now.

kyilmaz80 commented:
The [s3](http://bbuseruploads.s3.amazonaws.com/) may not be accessible from some locations. On the other hand, when I change the url the fix worked for me. I was able to build the libvirt box. In addtion only vagrant box file build. I had to create qemu qcow2 file from this box file.

Jopinder reviewed (changes requested):
The file should be renamed to openjdk7.bat and packer/templates/windows_2008_r2.json line 290 updated as well. 

scripts/configs/configure_firewall.bat line 7 should also reflect the change from OpenJDK 6 to OpenJDK 7

parzival0037 committed:
Rename install_openjdk6.bat to install_openjdk7.bat

ShahzebFarruk commented:
@Jopinder Please see the suggested changes made and pull requests created for the mentioned files. #599  #600 

 i don't get much time to review entire repo. Thanks for the suggesations! ;)

jmartin-r7 commented:
This project builds an intentionally vulnerable VM that intentionally utilized outdated software to demonstrate and practice the security testing.  I see that the original link is dead, however the project should not update the jdk version as that would possibly remove some of the avenues for attack of this target. If the file is available from another trusted source and can be verified by checksum, it the target location would simply be updated.  If the original file cannot be linked any update should be as minimal as possible to and each published known attack vector for this target needs to validated as continuing to function.

parzival0037 committed:
Updated broken link and rename install_openjdk7.bat to install_openjdk6.bat

updated the broken link in the issue #595 as per recommendations by @jmartin-tech

ShahzebFarruk commented:
@jmartin-tech, makes sense. Based upon reasoning, keeping the jdk to same version. I updated the broken link and applied. Please check.

jmartin-tech committed:
additional update for openjdk

ifteneiftene commented on scripts/installs/install_openjdk6.bat:
```diff
@@ -1,2 +1,2 @@
-powershell -Command "[System.Net.ServicePointManager]::SecurityProtocol = [System.Net.SecurityProtocolType]::Tls12; (New-Object System.Net.WebClient).DownloadFile('https://github.com/downloads/alexkasko/openjdk-unofficial-builds/openjdk-1.6.0-unofficial-b27-windows-amd64.zip', 'C:\Windows\Temp\openjdk-1.6.0-unofficial-b27-windows-amd64.zip')" <NUL
-cmd /c ""C:\Program Files\7-Zip\7z.exe" x "C:\Windows\Temp\openjdk-1.6.0-unofficial-b27-windows-amd64.zip" -oC:\openjdk6"
+powershell -Command "[System.Net.ServicePointManager]::SecurityProtocol = [System.Net.SecurityProtocolType]::Tls12; (New-Object System.Net.WebClient).DownloadFile('https://github.com/downloads/alexkasko/openjdk-unofficial-builds/openjdk-1.6.0-unofficial-b28-windows-amd64-installer.zip', 'C:\Windows\Temp\openjdk-1.6.0-unofficial-b28-windows-amd64-installer.zip')" <NUL
```
Waaa
